src/main/groovy/org/boozallen/plugins/jte/init/primitives/ReservedVariableName.groovy

```diff
@@ -34,4 +30,8 @@ abstract class ReservedVariableName implements ExtensionPoint{
         return Jenkins.get().getExtensionList(ReservedVariableName)
     }
 
+    abstract String getName()
```

Comment:
seems to be generally used as ..
String oopsMsg = String.format("Variable name %s is reserved for steps to access their library configuration", config);
            throw new Exception(oopsMsg);

Thinking ...
add a getMessageFormat() for the specific message that wraps the variable name,
make the ReservedVariableName::throw*LockExceptions ->

String oopsMsg = String.format(getMessageFormat(), getName());
throw new Exception(oopsMsg);